cmd2/cmd2.py

```diff
@@ -2391,7 +2396,7 @@ def _alias_delete(self, args: argparse.Namespace) -> None:
             self.aliases.clear()
             self.poutput("All aliases deleted")
         elif not args.name:
-            self.do_help('alias delete')
+            self.perror("Either --all or alias name(s) must be specified")
```

Comment:
Is there a way to enforce things to be correct at the argparse level so that this check is unnecessary?

Reply:
None that I know of